pkg/certs/certgen.go

```diff
@@ -151,3 +244,17 @@ func serviceNames(service, namespace string) []string {
 		fmt.Sprintf("%s.%s.svc.cluster.local", service, namespace),
```

Comment:
These assume a cluster dns of `cluster.local`, should we add more config for that?